README.rst

```diff
@@ -33,7 +33,7 @@ For that, it gives you a class decorator and a way to declaratively define the a
 
 .. -code-begin-
 
-.. code-block:: pycon
+.. code-block:: python
```

Comment:
`pycon` is actually a supported [lexer](https://pygments.org/docs/lexers/#pygments.lexers.python.PythonConsoleLexer), short for "python console". It'd be a cute easter egg though...

Reply:
Thanks for that! I learned something new!